### PR TITLE
EDITS: Minor edits made to content in run-access-cm.md

### PR DESCRIPTION
**Pull request by heidinett**
Made some minor edits to ARE/Gadi workflow in run-access-cm.md

Fixes Issue #587

### CODE REVIEW

File: docs/models/run-a-model/run-access-cm.md [hunk 1]
```diff
@@ -18,12 +18,12 @@
...
 <div class="note">
-    The workflow to run ACCESS_CM is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
+    The workflow to run ACCESS-CM is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
     <br>
-    The tabs above let you choose the type of workflow you would like to follow.
+    The above tabs allow you choose the type of workflow you would prefer to follow.
     <br>
     <br>
-    If you are new to ACCESS-CM, we <b>strongly</b> suggest you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be interrupted.
+    If you are new to ACCESS-CM, we <b>recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:26]:
```suggestion
    If you are new to ACCESS-CM, we <b>strongly recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.
```

File: docs/models/run-a-model/run-access-cm.md [hunk 2]
```diff
@@ -18,12 +18,12 @@
 <!-- End of tab content -->
 
 <div class="note">
-    The workflow to run ACCESS_CM is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
+    The workflow to run ACCESS-CM is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:21]:
```suggestion
    The workflow to run {{ model }} is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
```

File: docs/models/run-a-model/run-access-cm.md (hunk 1 above)
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:26]:
```suggestion
    If you are new to {{ model }}, we <b>recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.
```

File: docs/models/run-a-model/run-access-cm.md [hunk 3]
```diff
@@ -115,42 +115,40 @@ Before running {{ model }}, you need to fulfil general prerequisites outlined in
...
                 <br>
-                This is the list (joined by <code>+</code> signs) of project data storage needed for the {{ model }} simulation. In ARE, storage locations need to be explicitly defined to access data from within a VDI instance.
+                This is a list of all project data storage (joined by <code>+</code> signs) needed for the {{ model }} simulation. In ARE, storage locations need to be explicitly defined to access data from within a VDI instance.
                 <br>
-                Since every {{ model }} simulation can be unique and input data can come from various sources, if your specific simulation requires data coming from projects other than <code>access</code>, <code>hh5</code>, <code>hr22</code> or <code>ki32</code>, you need to add those projects to the storage path.
+                An {{ model }} simulation can be unique and input data can originate from various sources. Hence, if your simulation requires data originating from projects other than <code>access</code>, <code>hh5</code>, <code>hr22</code> or <code>ki32</code>, you need to add those projects to the storage path.
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:129]:
```suggestion
                Every {{ model }} simulation can be unique and input data can originate from various sources. Hence, if your simulation requires data stored in project folders other than <code>access</code>, <code>hh5</code>, <code>hr22</code> or <code>ki32</code>, you need to add those projects to the storage path.
```

File: docs/models/run-a-model/run-access-cm.md [hunk 4]
```diff
@@ -115,42 +115,40 @@ Before running {{ model }}, you need to fulfil general prerequisites outlined in
...
                 <br>
-                For example, if your {{ model }} simulation requires data coming from <code>/g/data/tm70</code> and <code>/scratch/w40</code>, your full storage path will be: <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32<b>+/gdata/tm70+scratch/w40</b></code>
+                For example, if your {{ model }} simulation requires data from <code>/g/data/tm70</code> and <code>/scratch/w40</code>, your full storage path will be: <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32<b>+/gdata/tm70+scratch/w40</b></code>
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:131]:
```suggestion
                For example, if your {{ model }} simulation requires data stored in <code>/g/data/tm70</code> and <code>/scratch/w40</code>, your full storage path will be: <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32<b>+/gdata/tm70+scratch/w40</b></code>
```

File: docs/models/run-a-model/run-access-cm.md [hunk 5]
```diff
@@ -115,42 +115,40 @@ Before running {{ model }}, you need to fulfil general prerequisites outlined in
...
             </li>
         </ul>
-        Launch the session and, after the session starts, click on <i>Launch VDI Desktop</i>.
+        Now, launch an ARE session and, once it starts, click on <i>Launch VDI Desktop</i>.
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:134]:
```suggestion
        Launch the ARE session and, once it starts, click on <i>Launch VDI Desktop</i>.
```

File: docs/models/run-a-model/run-access-cm.md [hunk 6]
```diff
@@ -83,13 +83,13 @@ Before running {{ model }}, you need to fulfil general prerequisites outlined in
     <!-- ARE/Gadi-->
     <div>
         <div class="note">
-            Your chosen workflow is ARE / <i>Gadi</i>. If you want to run {{ model }} on <i>accessdev</i> instead, please select <a href="#accessdev"><i>accessdev</i> workflow</a>.
+            Your chosen workflow is ARE/<i>Gadi</i>. If you would prefer to run {{ model }} on <i>accessdev</i>, then select the <a href="#accessdev"><i>accessdev</i> workflow</a>.
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:86]:
```suggestion
            Your chosen workflow is ARE / <i>Gadi</i>. If you would prefer to run {{ model }} on <i>accessdev</i>, then select the <a href="#accessdev"><i>accessdev</i> workflow</a>.
```

I think leaving space makes it more readabale. Similarly to what is done in other similar parts of the page.

File: docs/models/run-a-model/run-access-cm.md [hunk 7]
```diff
@@ -115,42 +115,40 @@ Before running {{ model }}, you need to fulfil general prerequisites outlined in
...
     <div>
         <div class="note">
-            Your chosen workflow is <i>accessdev</i>. If you want to run {{ model }} on ARE / <i>Gadi</i> instead, please select <a href="#are">ARE / <i>Gadi</i> workflow</a>.
+            Your chosen workflow is <i>accessdev</i>. If you would prefer to run {{ model }} on ARE / <i>Gadi</i>, then select the <a href="#are">ARE / <i>Gadi</i> workflow</a>.
         </div>
         Login to <i>accessdev</i> by runnning:
         <pre><code>ssh accessdev</code></pre>
         <div class="note">
-            If you have not yet set up your <i>accessdev</i> connection through `ssh`, please check the <a href="https://accessdev.nci.org.au/trac/wiki/GettingConnected">Getting Connected to Accessdev</i></a> guide.
+            If you have not yet setup your <i>accessdev</i> connection through `ssh`, please check the <a href="https://accessdev.nci.org.au/trac/wiki/GettingConnected">Getting Connected to Accessdev</i></a> guide.
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:151]:
```suggestion
            If you have not yet set up your <i>accessdev</i> connection through `ssh`, please check the <a href="https://accessdev.nci.org.au/trac/wiki/GettingConnected">Getting Connected to Accessdev</i></a> guide.
```

I think the verb is **set up**, not to be confused with the noun/adjective **setup** (ref. --> https://www.future-perfect.co.uk/grammar-tips/easy-mistakes/is-it-setup-set-up-or-set-up).

File: docs/models/run-a-model/run-access-cm.md [hunk 8]
```diff
@@ -219,7 +217,7 @@ module load cylc7</code></pre>
                 <br>
                 <ul>
                     <li>
-                        <i>Local-only copy</i>
+                        <i>Local copy</i>
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-cm.md:220]:
```suggestion
                        <i>Local-only copy</i>
```
I think it needs to be stressed that is only a local copy.

**Review by atteggiani — CHANGES_REQUESTED** [review]

File: docs/models/run-a-model/run-access-cm.md (hunk 7 above)
Comment (heidinett) [reply docs/models/run-a-model/run-access-cm.md:151]: ARE/Gadi was suggested for consistentcy with other uses of this term on the page. But, I am fine with your suggestion.

**Review by atteggiani — APPROVED** [review]

**Pull request merged**